src/main/java/com/powsybl/openloadflow/ac/solver/NewtonRaphson.java

```diff
@@ -84,23 +83,32 @@ public void reportAndLogLargestMismatchByAcEquationType(Reporter reporter, Equat
                         int busPhiRow = equationSystem.getVariable(elementNum, AcVariableType.BUS_PHI).getRow();
                         double busV = equationSystem.getStateVector().get(busVRow);
                         double busPhi = equationSystem.getStateVector().get(busPhiRow);
-                        LOGGER.trace("Mismatch `{}` for {}: {} (element={}) || Bus V /_ PHI = {} /_ {}", acEquationType, equation, equationMismatch, elementId, busV, busPhi);
-                        if (iterationMismatchReporter != null) {
-                            Reports.reportNewtonRaphsonMismatch(iterationMismatchReporter, getEquationTypeDescription(acEquationType), equationMismatch, elementId, busV, busPhi, iteration);
+                        LfBus bus = network.getBus(elementNum);
+                        double busNominalV = bus.getNominalV();
+                        double busSumP = bus.getP().eval() * PerUnit.SB;
+                        double busSumQ = bus.getQ().eval() * PerUnit.SB;
+
+                        LOGGER.trace("Mismatch {} on {}: {}", acEquationType, equation, equationMismatch);
+                        LOGGER.trace("    Bus        Id       : {}", elementId);
+                        LOGGER.trace("    Bus nominal V [  kV]: {}", busNominalV);
+                        LOGGER.trace("    Bus         V [p.u.]: {}", busV);
+                        LOGGER.trace("    Bus       Phi [ rad]: {}", busPhi);
+                        LOGGER.trace("    Bus     sum P [  MW]: {}", busSumP);
+                        LOGGER.trace("    Bus     sum Q [MVar]: {}", busSumQ);
```

Comment:
should check if trace logging enabled for this block

change `Mismatch` to `Largest mismatch on <getEquationTypeDescription(acEquationType)>`

for logging purposes we can limit to Vmag and Vangle, no need for the others, already in the report (this is too many lines here, as it was before was better)